Mesh_3/include/CGAL/Mesh_3/Protect_edges_sizing_field.h

```diff
@@ -455,12 +473,32 @@ class Protect_edges_sizing_field
     return s;
   }
 
+  /// Query the sizing field and return its value at the point `p`, or
+  /// `minimal_size` if the latter is greater.
+  FT query_size(const Bare_point& p, int dim, const Index& index) const
+  {
+    FT s = query_field(p, dim, index, size_);
+
+    // if(minimal_size_ != FT() && s < minimal_size_)
```

Comment:
So, this commented section is no longer needed?

Reply:
Yes, it is no longer needed. The check was moved to the function refine_balls at to line 1477.